packages/headless/src/graphql.module.dependency.ts

```diff
@@ -0,0 +1,17 @@
+import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo';
+import { GraphQLModule } from '@nestjs/graphql';
+import { HelperModule } from './helper/helper.module';
+import { WishListModule as WishListGraphqlModule } from './modules/wishlist/wishlist.graphql.module';
+
+export const ResolveGraphqlModule = () => {
+  return [
+    HelperModule,
+    WishListGraphqlModule,
+    GraphQLModule.forRoot<ApolloDriverConfig>({
+      driver: ApolloDriver,
+      typePaths: ['src/**/*.graphql'],
+      path: '/v1/graphql',
```

Comment:
We dont need version for now. keep the default. Also playground value will be dependent on env

Reply:
ok